firecracker-control/local.go

```diff
@@ -48,25 +45,31 @@ func init() {
 	})
 }
 
-// TODO: Remove this interface after merging PR #91
-// See https://github.com/firecracker-microvm/firecracker-go-sdk/pull/91
-type machineIface interface {
-	Start(context.Context) error
-	StopVMM() error
-	Wait(context.Context) error
-	SetMetadata(context.Context, interface{}) error
+const (
+	startEventName = "/vm/start"
```

Comment:
nit: may want to be a little more specific, i.e. `/firecracker-vm/start`, unless containerd has officially defined this is a topic to publish to for any VM manager shim.

Reply:
I couldn't find any requirements for topics, so I think its fine to prefix them `/firecracker-vm`